events/index.html

```diff
@@ -1,9 +1,7 @@
 {% extends "layout-side-nav.html" %}
 {% import "_vars-events.html" as vars with context %}
 
-{# This sets the related links in the footer
-{% set view = get_document('views', 'newsroom') %}
-#}
+{% set view = get_document('views', 'blog') %}
```

Comment:
Yes, but since the events view doesn't exist and newsroom didn't have what I needed, I was testing this with the blog view. Reverted in 413c3fb